src/main/java/frc/robot/commands/AutoIntakeCommand.java

```diff
@@ -5,18 +5,21 @@
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.subsystems.CrocodileSubsystem;
+import java.util.function.BooleanSupplier;
 
 public class AutoIntakeCommand extends CommandBase {
     private final CrocodileSubsystem crocodileSubsystem;
     private final double speed;
     private final Timer timer;
     private final Debouncer debouncer;
+    private final BooleanSupplier stopIntake;
 
-    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed) {
+    public AutoIntakeCommand(CrocodileSubsystem crocodileSubsystem, double speed, BooleanSupplier stopIntake) {
```

Comment:
Could this possibly be an overload of the constructor instead?

Reply:
Yes I can change that